config/secrets.yml

```diff
@@ -30,3 +34,5 @@ test:
 
 production:
   secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
+  slack_token: "xoxp-235986621104-238493557106-238649524195-b886dacfa83620d7c79b483114198cf9"
+  slack_channel: "#shelters"
```

Comment:
This is gonna get way too noisy. We need a different channel for this.

Reply:
Thanks, I created `shelters_notif` for this purpose.